lib/tests/sources.sh

```diff
@@ -9,17 +10,18 @@ die() {
 }
 
 if test -n "${TEST_LIB:-}"; then
-  export NIX_PATH=nixpkgs="$(dirname "$TEST_LIB")"
+  NIX_PATH=nixpkgs="$(dirname "$TEST_LIB")"
 else
-  export NIX_PATH=nixpkgs="$(cd $(dirname ${BASH_SOURCE[0]})/../..; pwd)"
+  NIX_PATH=nixpkgs="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.."; pwd)"
```

Comment:
Isn't this triggering the warning that cd might fail and ``|| exit`` should be added?

Reply:
I don't understand. `shellcheck lib/tests/sources.sh` doesn't show that warning, and `set -o errexit` + `shopt -s inherit_errexit` ensures that `|| exit` is unnecessary.

Reply:
I mean https://github.com/koalaman/shellcheck/wiki/SC2164

Reply:
I know that one, but what do you mean? Are you seeing an error? If so, can you please show that?